notebooks/00_quick_start/naml_synthetic.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>it would be great to add also the integration tests here. Jianxun added some for the SLiRec notebook, see here: <a href="https://github.com/microsoft/recommenders/blob/master/tests/integration/test_notebooks_gpu.py#L239" target="_blank">https://github.com/microsoft/recommenders/blob/master/tests/integration/test_notebooks_gpu.py#L239</a></p><p> </p><p>See instructions here <a href="https://github.com/microsoft/recommenders/tree/master/tests#developing-smoke-and-integration-tests-with-papermill" target="_blank">https://github.com/microsoft/recommenders/tree/master/tests#developing-smoke-and-integration-tests-with-papermill</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>